cl/users/models.py

```diff
@@ -360,10 +344,101 @@ class Meta:
             models.Index(fields=["message_id"]),
         ]
 
+    def convert_to_email_multipart(
+        self,
+    ) -> EmailMessage | EmailMultiAlternatives:
+        """Composes an email multipart message from the data stored.
+
+        :return email: Return an EmailMessage or EmailMultiAlternatives message
+        """
+
+        # Get the required fields to compose the email multipart message
+        keys_to_use = ("subject", "from_email", "reply_to", "headers", "to")
+        msg_dict = {k: v for k, v in self.__dict__.items() if k in keys_to_use}
+
+        # If the message has a related User, choose the user email address to
+        # ensure we send it to the updated user email address.
+        if self.user:
+            msg_dict["to"] = [self.user.email]
+
+        email: EmailMessage | EmailMultiAlternatives
+        if self.html_message:
+            if self.plain_text:
+                msg_dict["body"] = self.plain_text
+                email = EmailMultiAlternatives(**msg_dict)
```

Comment:
I couldn't pull this line out because depending on the availability of email content (plain_text or html), we need to assign it to the dict key `body` before and then create the `EmailMultiAlternatives`